packages/suite/src/components/suite/modals/AddAccount/components/AddAccountButton.tsx

```diff
@@ -13,6 +13,13 @@ const verifyAvailability = ({
     emptyAccounts: Account[];
     account: Account;
 }) => {
+    if (!account) {
```

Comment:
This is bad you cannot conditionally use hooks (useCallback) here. Please move ifs after that.

Reply:
here is no conditionally used hook, or I (and eslint) can't see it. This is separate `verifyAvailability` function, not render fuction of `AddDefaultAccountButton`...

Reply:
I see, it looked like that because of github diff... sorry for bothering